File: problemas/matematica/1761/1761.dart
import 'dart:io';
import 'dart:math';

void main() {
  while (true) {
    String? linha = stdin.readLineSync();
    if (linha == null) {
        break;
    }

    List<double> entrada = linha.split(' ').map((x) => double.parse(x)).toList();
    double a = entrada[0];
    double b = entrada[1];
    double c = entrada[2];
    
    print((5.0 * (tan(a * pi / 180.0) * b + c)).toStringAsFixed(2));
  }
}

File: problemas/matematica/1761/1761.jl
using Printf

pi = 3.141592654
while !eof(stdin)
  entrada = [parse(Float64, x) for x in split(readline())]
  a = entrada[1]
  b = entrada[2]
  c = entrada[3]

  @printf "%.2f\n" (5.0 * (tan(a * pi / 180.0) * b + c))
end